runtime/turing/src/lib.rs

```diff
@@ -1241,7 +1242,14 @@ impl_runtime_apis! {
 					u32::from(para_id),
 					CurrencyId::from(currency_id),
 					encoded_call_weight,
-				).map_err(|_| "cannot get xcmp fee")?.0;
+				).map_err(|e| {
+					match e {
+						DispatchError::Module(ModuleError{
+							message: Some("CurrencyChainComboNotFound"), ..
+						})=> "currency chain combination not supported",
+						_ => "cannot get xcmp fee"
+					}
+				})?.0;
 				let inclusion_fee = TransactionPayment::query_fee_details(extrinsic, len)
```

Comment:
currency/chain combo error is important enough to pass through. Are there others? Is there a better way to do this?

Reply:
these are the other two:
```
		/// Either the weight or fee per second is too large.
		FeeOverflow,
		/// Either the instruction weight or the transact weight is too large.
		WeightOverflow,

Reply:
Updated to forward all module errors through.